CHANGES.md

```diff
@@ -1,6 +1,473 @@
+  - moved Chapel's website from http://chapel.cray.com to https://chapel-lang.org
+  - added a color-coded version selection menu to the online documentation
+
+File / Directory Structure
+--------------------------
+* made `make install` install mason
+* added the CHPL_COMM_SUBSTRATE setting to GASNet-specific paths
+
+New Tools / Tool Changes
+------------------------
+* added support for bash autocompletion for `chpl` compiler options
+  (see https://chapel-lang.org/docs/1.17/usingchapel/compiling.html#tab-autocompletion-for-flags)
+* changed the generated executable's name from the main module to its filename
+  (see the `-o` flag on https://chapel-lang.org/docs/1.17/usingchapel/man.html)
+* added support for multiple mason repositories, including local registries
+  (see https://chapel-lang.org/docs/1.17/tools/mason/mason.html#local-registries)
+* added support for `mason help` and `mason version` commands
+* `chpl` now offers suggestions for unfamiliar flags
+  (see `chpl -fast`)
+* rewrote `printchplenv` and reworked its interface and output
+  (see `util/printchplenv --help`)
+* rewrote `compileline`
+  (see `util/config/compileline --help`))
+
+Semantic Changes / Changes to Chapel Language
+---------------------------------------------
+* `in` intents now behave more like variable initialization
+  (see https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/chips/13.rst)
+* ranges are now passed by `const in` intent by default
+  (see 'The Default Intent' in the 'Procedures' chapter of the specification)
+* disallowed implicit coercions for type arguments with a formal type specifier
+  (see 'Implicit Conversions' in the specification's 'Conversions' chapter)
+* changed the alignment of a non-stridable range from 0 to its low bound
+  (i.e., `(3..5).alignment` is now 3 where it used to be 0)
+
+New Features
+------------
+* added support for uninterpreted triple-quoted strings
+  (see 'Literals' section in 'Lexical Structure' chapter of the specification)
+* added support for `continue` statements within forall loops
+* extended `delete` to accept arrays and list of expressions to delete
+  (see 'Deleting Class Objects' in the 'Classes' chapter of the specification)
+* added an `isEmpty()` method on domains
+* type and param fields can now be accessed from a type variable
+  (see 'Field Accesses' in the 'Classes' chapter of the specification)
+* supported forwarding method calls on arrays, domains, and distributions
+  (see https://chapel-lang.org/docs/1.17/technotes/dsi.html#overview)
+* added support for associative domains whose index types contain ranges
+* added support for subtype queries on distributions
+  (see 'Methods on All Domain Types' in the 'Domains' chapter of the spec)
+* added support for using GASNet's `smp` conduit
+
+Feature Improvements
+--------------------
+* dramatically improved support for user-defined initializers
+  (see https://chapel-lang.org/docs/1.17/technotes/initializers.html)
```

Comment:
I don't know if this is the right documentation URL or if it just needs to be updated today. It describes the old system (before CHIP 23).

Reply:
We're working on it